### PR TITLE
Update BN and QAT listing

### PR DESCRIPTION
**Pull request by Xinnoh**
confirmed spanish with yuii-
Also updated the process of applying for bn in the English version.

### CODE REVIEW

File: wiki/People/Beatmap_Nominators/en.md [hunk 1]
```diff
@@ -32,24 +32,23 @@ Modders that aim to apply to the Beatmap Nominators must fulfil the following cr
 + The point in time at which the Kudosu has been awarded to you mod post does not matter. The only relevant information about a mod post is its post date and time in UTC+0 as well as the fact that it has been awarded Kudosu.
 
 ### How the Required Minimum Score is Calculated
-+ This depends on the total `Kudosu Score` and the total `Mod Score` an applicant achieved over the last months. A mathematical formula is used to determine these values.
-+ `Kudosu Score` is determined according to the following formula:
-
-![Kudosu Score Formula](/wiki/shared/KDSCORE.png "Kudosu Score Formula")
++ This depends on the total `Mod Score` an applicant achieved over the last months. This value is determined according to the following formula:
```
Comment (MegaApplePi) [inline wiki/People/Beatmap_Nominators/en.md]: Where did the formula go? Shouldn't the last sentence say `A mathematical formula is used to determine these values.`?

File: wiki/People/Beatmap_Nominators/en.md [hunk 2]
```diff
@@ -32,24 +32,23 @@ Modders that aim to apply to the Beatmap Nominators must fulfil the following cr
...
 
-+ `Mod Score` per month is determined according to the following formula:
++ `Kudosu Score` is determined according to the following formula:
```
Comment (MegaApplePi) [inline wiki/People/Beatmap_Nominators/en.md]: Kudosu Score or Mod Score?

File: wiki/People/Beatmap_Nominators/en.md [hunk 3]
```diff
@@ -32,24 +32,23 @@ Modders that aim to apply to the Beatmap Nominators must fulfil the following cr
...
++ m = Minimum expected number of individual beatmaps modded in a month
++ M = Total achieved number of individual beatmaps modded in a month
 
-![Mod Score Formula](/wiki/shared/MODSCORE.png "Mod Score Formula")
+![Kudosu Score Formula](/wiki/shared/Modscore_new.png "Kudosu Score Formula")
```
Comment (MegaApplePi) [inline wiki/People/Beatmap_Nominators/en.md]: Kudosu Score or Mod Score?

Comment (Xinnoh) [reply wiki/People/Beatmap_Nominators/en.md]: fixed the stuff

File: wiki/People/Beatmap_Nominators/en.md [hunk 4]
```diff
@@ -34,11 +34,11 @@ Modders that aim to apply to the Beatmap Nominators must fulfil the following cr
 ### How the Required Minimum Score is Calculated
 + This depends on the total `Mod Score` an applicant achieved over the last months. This value is determined according to the following formula:
 
-+ `Kudosu Score` is determined according to the following formula:
+![Mod Score Formula](/wiki/shared/Modscore_new.png "Kudosu Score Formula")
```
Comment (MegaApplePi) [inline wiki/People/Beatmap_Nominators/en.md]: the image should use white text instead of black

Comment (Xinnoh) [reply wiki/People/Beatmap_Nominators/en.md]: idk how to do this aaa

**Comment by cl8n** [comment]
yeah it's fine to not do them here. just make sure they have the outdated tag if it's not there already

File: wiki/People/Beatmap_Nominators/en.md [hunk 5]
```diff
@@ -75,55 +74,66 @@ _Please note: All BN speak English unless otherwise noted._ Link to the [user gr
...
+| [squirrelpascals](https://osu.ppy.sh/users/6151332)   |                          |
 | [Sonnyc](https://osu.ppy.sh/users/11771)              | Korean                   |
+| [Sotarks](https://osu.ppy.sh/users/4452992)           | French                   |
 | [TheKingHenry](https://osu.ppy.sh/users/5128277)      | Finnish                  |
 | [timemon](https://osu.ppy.sh/users/2072005)           | Thai                     |
 | [Trynna](https://osu.ppy.sh/users/2652951)            | Portuguese               |
 | [Ultima Fox](https://osu.ppy.sh/users/3198109)        |                          |
 | [UndeadCapulet](https://osu.ppy.sh/users/2523533)     |                          |
+| [Uta](https://osu.ppy.sh/users/9439796)               | Indonesian               |
 | [Wishkey](https://osu.ppy.sh/users/1101704)           | Dutch, German            |
```
Comment (Raidencio) [inline wiki/People/Beatmap_Nominators/en.md]: wishkey resigned some days ago

File: wiki/People/Beatmap_Nominators/en.md [hunk 6]
```diff
@@ -75,55 +74,66 @@ _Please note: All BN speak English unless otherwise noted._ Link to the [user gr
...
+| [yaspo](https://osu.ppy.sh/users/4945926)             |                          |
 | [Yauxo](https://osu.ppy.sh/users/898306)              | German                   |
+| [Yukiyo](https://osu.ppy.sh/users/4541873)            | German                   |
 
 ### osu!taiko
 
 | Name                                                  | Additional Languages     |
 |-------------------------------------------------------|--------------------------|
-| [ARGENTINE DREAM](https://osu.ppy.sh/users/1981781)   | Spanish                  |
+| [ARGENTINE DREAM](https://osu.ppy.sh/users/1981781)   | Portuguese, Spanish, Japanese|
```
Comment (Raidencio) [inline wiki/People/Beatmap_Nominators/en.md]: add Catalan for ARGENTINE DREAM

File: wiki/People/Beatmap_Nominators/en.md [hunk 7]
```diff
@@ -133,30 +143,39 @@ _Please note: All BN speak English unless otherwise noted._ Link to the [user gr
...
-| [Yales](https://osu.ppy.sh/users/2377881)             | French                   |
+| [Yales](https://osu.ppy.sh/users/2377881)             | French, Japanese         |
 
 ### osu!catch
 
 | Name                                                  | Additional Languages     |
 |-------------------------------------------------------|--------------------------|
 | [alienflybot](https://osu.ppy.sh/users/636114)        | Cantonese, Chinese       |
 | [Ascendance](https://osu.ppy.sh/users/2931883)        |                          |
+| [Ajamez](https://osu.ppy.sh/users/7890134)            | Filipino                 |
```
Comment (Raidencio) [inline wiki/People/Beatmap_Nominators/en.md]: AJamez (cap the J)

File: wiki/People/Beatmap_Nominators/en.md [hunk 8]
```diff
@@ -75,55 +74,66 @@ _Please note: All BN speak English unless otherwise noted._ Link to the [user gr
...
 | [Net0](https://osu.ppy.sh/users/5099768)              | Portuguese               |
 | [Nevo](https://osu.ppy.sh/users/7451883)              |                          |
-| [Noffy](https://osu.ppy.sh/users/1541323)             |                          |
 | [Nozhomi](https://osu.ppy.sh/users/2716981)           | French                   |
 | [Okoratu](https://osu.ppy.sh/users/1623405)           | German                   |
 | [pimpG](https://osu.ppy.sh/users/328615)              | Portuguese               |
-| [Plaudible](https://osu.ppy.sh/users/7149815)         |                          |
 | [Realazy](https://osu.ppy.sh/users/918297)            | French                   |
-| [Regraz](https://osu.ppy.sh/users/3076909)            | Chinese                  |
 | [riffy](https://osu.ppy.sh/users/597957)              | Russian                  |
```
Comment (Raidencio) [inline wiki/People/Beatmap_Nominators/en.md]: riffy resigned a long time ago

File: wiki/People/Beatmap_Nominators/en.md [hunk 9]
```diff
@@ -75,55 +74,66 @@ _Please note: All BN speak English unless otherwise noted._ Link to the [user gr
...
 | [Gero](https://osu.ppy.sh/users/1467715)              | Spanish                  |
-| [Gottagof4st](https://osu.ppy.sh/users/6122935)       | Dutch                    |
+| [Hailie](https://osu.ppy.sh/users/5226970)            |                          |
+| [hypercyte](https://osu.ppy.sh/users/9155377)         |                          |
 | [IceKalt](https://osu.ppy.sh/users/5410645)           | German                   |
 | [iYiyo](https://osu.ppy.sh/users/3919785)             | Spanish, some German     |
 | [jonathanlfj](https://osu.ppy.sh/users/270377)        | Chinese, some French     |
+| [Kaitjuh](https://osu.ppy.sh/users/2225327)           | Dutch                    |
 | [Kalibe](https://osu.ppy.sh/users/3376777)            | Polish                   |
 | [Kibbleru](https://osu.ppy.sh/users/3193504)          |                          |
```
Comment (Raidencio) [inline wiki/People/Beatmap_Nominators/en.md]: Kibbleru is QAT now

**Comment by sr229** [comment]
If this is still a approved pending merge, you might wanna fix the conflicts

**Comment by Xinnoh** [comment]
ok everything is now fixed

File: wiki/People/Quality_Assurance_Team/en.md [hunk 10]
```diff
@@ -15,26 +15,31 @@ Team Members
...
 | [Deif](https://osu.ppy.sh/users/318565)               | ![No][false] | ![No][false] | ![Yes][true] | ![No][false] | Spanish, German         |
 | [Feerum](https://osu.ppy.sh/users/4815717)            | ![No][false] | ![No][false] | ![No][false] | ![Yes][true] | German, Polish          |
 | [Fycho](https://osu.ppy.sh/users/1876867)             | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | Chinese                 |
-| [Gabe](https://osu.ppy.sh/users/654108)             | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | French                     |
-| [Hobbes2](https://osu.ppy.sh/users/8157492)             | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] |                        |
+| [Gabe](https://osu.ppy.sh/users/654108)               | ![Yes][true] | ![Yes][true] | ![No][false] | ![No][false] | French                  |
+| [Gottagof4st](https://osu.ppy.sh/users/6122935)       | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | Dutch                   |
+| [Hobbes2](https://osu.ppy.sh/users/8157492            | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] |                         |
```
Comment (WalterToro) [inline wiki/People/Quality_Assurance_Team/en.md]: Missing ')' here

**Pull request merged**